dev/release/05-binary-upload.sh

```diff
@@ -21,8 +21,17 @@ set -e
 set -u
 set -o pipefail
 
-export LANG=C
-export LC_CTYPE=C
+export LANG=C.UTF-8
+export LC_ADDRESS=C.UTF-8
+export LC_CTYPE=C.UTF-8
+export LC_IDENTIFICATION=C.UTF-8
+export LC_MEASUREMENT=C.UTF-8
+export LC_MONETARY=C.UTF-8
+export LC_NAME=C.UTF-8
+export LC_NUMERIC=C.UTF-8
+export LC_PAPER=C.UTF-8
+export LC_TELEPHONE=C.UTF-8
+export LC_TIME=C.UTF-8
```

Comment:
You could set `LC_ALL=C.UTF-8`

Reply:
Ah, I misunderstood `LC_ALL`. It can be overwritten by other `LC_*` like `LANG`.
I'll simplify the code. Thanks.

Reply:
#41247